wiki/Help_centre/Beatmapping/de.md

```diff
@@ -0,0 +1,51 @@
+---
+tags:
+  - upload
+  - delete
+  - beatmap
+  - slot
+  - editor
+  - status
+  - warning
+  - hochladen
+  - Warnung
+  - löschen
+---
+
+# Mapping und Editor
+
+*Hauptseite: [Hilfecenter](/wiki/Help_centre)*
+
+Dieser Abschnitt beschäftigt sich mit Problemen mit dem Beatmap-Editor und Mapping generell.
+
+## Wie lade ich meine Beatmap hoch oder aktualisiere sie? {#upload-beatmap}
+
+**Du kannst das über den Editor im Spiel machen.**
+
+Gehe in den Editor und klicke auf `Datei` > `Beatmap hochladen`, um deine Beatmap hochzuladen.
+
+Dadurch wird deine Beatmap auf der Webseite für andere verfügbar zum Herunterladen. Das Aktualisieren einer Beatmap ist derselbe Prozess für eine Beatmap, die bereits hochgeladen wurde.
+
+Jeder Nutzer hat eine begrenzte Anzahl an Beatmaps, die er gleichzeitig hochladen kann. Wenn eine Beatmap für einen Monat (28 Tage) lang inaktiv gelassen wird, indem sie nicht aktualisiert wird, wird sie auf den [Beatmap-Friedhof](/wiki/Beatmap/Category#graveyard) verschoben. Aktualisiere die Beatmap, um sie aus dem Beatmap-Friedhof zurück in den Status "Pending" oder "WIP" zu setzen.
+
+## Der Editor sagt mir "Du bist nicht der Besitzer der Beatmap, die du gerade hochladen möchtest!" {#wrong-author}
+
+**Das kann passieren, wenn der Ersteller der Beatmap nicht korrekt in den Beatmap-Dateien eingestellt wurde.**
+
+Klicke im Editor-Menü auf `Datei`, dann auf `.osu im Editor öffnen`. Überprüfe die Zeile `Creator:` unter der Sektion `Metadata` und stelle sicher, dass der Name genau deinem Benutzernamen entspricht, einschließlich Großschreibung.
+
+Wenn er dem nicht entspricht, ändere ihn manuell für jeden Schwierigkeitsgrad in deiner Beatmap, lade die Beatmap neu und du solltest in der Lage sein, die Beatmap einzureichen.
+
+Solltest du immer noch Probleme haben oder nach mehr Informationen suchen, lies bitte die [Seite über Probleme mit dem BSS](/wiki/Guides/BSS_Issues).
+
+## Ich bin ein osu!supporter, aber ich habe keine 20 ausstehenden Slots! {#upload-slots}
+
+**Die Zahl der verfügbaren Slots hängt von der Zahl der gerankten Beatmaps ab, die du hast. Für mehr Details, siehe [osu!supporter § Erhöhte Grenzwerte](/wiki/osu!supporter#erhöhte-grenzwerte).**
+
+## Wie kann ich meine Beatmap löschen? {#delete-beatmap}
+
+**Das [Support-Team](/wiki/People/The_Team/Account_support_team) nimmt keine Anfragen an, um hochgeladene Beatmaps manuell zu löschen.**
+
+Du kannst deine Beatmap selbst löschen, indem du wartest, dass sie auf dem Beatmap-Friedhof verfällt, was nach einem Monat (28 Tage) ohne Aktualisierungen passieren wird.
```

Comment:
`verfallen` is a very close translation to `to decay` but I think `landen` sounds more natural in this context. So after reconsideration, I suggest to use `dass sie auf dem Beatmap-Friedhof landet`.

Reply:
Sounds good!

Reply:
```suggestion
Du kannst deine Beatmap selbst löschen, indem du wartest, dass sie auf dem Beatmap-Friedhof landet, was nach einem Monat (28 Tage) ohne Aktualisierungen passieren wird.
```